lib/JsonGenerator.js

```diff
@@ -6,11 +6,17 @@
 
 const { ConcatSource } = require("webpack-sources");
 
-const stringifySafe = data =>
-	JSON.stringify(data).replace(
+const stringifySafe = data => {
+	const stringified = JSON.stringify(data);
```

Comment:
A better fix would be to return `"null"` when `data` is `undefined`.
All other inputs return a string.

Reply:
`JSON.stringify(undefined)` returns `undefined`.

Reply:
Absolutely. But since it is the only case where it does not returns a string, it is possible to bypass the call and hardcode the result.